crawl-ref/source/stairs.cc

```diff
@@ -655,7 +655,6 @@ void floor_transition(dungeon_feature_type how,
     if (!forced)
     {
         apply_barbs_damage();
```

Comment:
These brackets should be removed to fix the style code linting (you can do that automatically by running `util/unbrace`)